Makefile

```diff
@@ -8,7 +8,7 @@ all:
 	@echo "Please choose a task."
 
 test:
```

Comment:
we should add `test-with-coverage` for this. As a developer i would like to execute `make test` without the coverage

ping @greg0ire @core23 @Soullivaneuh